invokeai/app/modules/metadata.py

```diff
@@ -0,0 +1,180 @@
+import json
```

Comment:
This should probably be in the `/services` folder (though it does probably need some organization soon). `/modules` doesn't provide much information, since every file in Python is a module.

Reply:
done